mandrill-mock/server.py

```diff
@@ -47,6 +47,20 @@
         return json_response({'message': "subaccount created (this isn't the same response as mandrill)"})
 
 
+async def mandrill_sub_account_delete(request):
```

Comment:
don't think you need this, I only created this account while load testing the new arq.